app/src/main/java/com/example/grutor/Fragments/LessonsFragment.java

```diff
@@ -125,26 +132,13 @@ public void onClick(View view) {
             @Override
             public void onClick(View view) {
                 Toast.makeText(getApplicationContext(), "Call", Toast.LENGTH_SHORT).show();
-
             }
         });
-
     }
 
-    private void queryUsers() {
+    private void queryUsers() throws ParseException {
         ParseQuery<ParseUser> query = ParseQuery.getQuery(ParseUser.class);
-        query.setLimit(5);
-        query.findInBackground(new FindCallback<ParseUser>() {
-            @SuppressLint("NotifyDataSetChanged")
-            @Override
-            public void done(List<ParseUser> theseUsers, ParseException e) {
-                if (e != null) {
-                    Log.e("Lessons Fragment", "Issue with getting lessons", e);
-                    return;
-                }
-                users.addAll(theseUsers);
-            }
-        });
+        users.addAll(query.find());
```

Comment:
looks like you change to synchronized query. but be careful it will block the UI if it take too long to load.